gossip-base/src/main/java/org/apache/gossip/manager/GossipManager.java

```diff
@@ -46,14 +48,21 @@
 public abstract class GossipManager {
 
   public static final Logger LOGGER = Logger.getLogger(GossipManager.class);
+  
+  // this mapper is used for ring and user-data persistence only. NOT messages.
+  public static final ObjectMapper metdataObjectMapper = new ObjectMapper() {{
```

Comment:
The "user" has access to the implementation choices by using the builder. Everything else here is an instance variable and I do not see a good reason for this particular thing to be static, especially public static if something down the line needs access to this I would rather pass references with getters then make it public static. Let me know if you do not think this is agreeable